docs/LoaderDebugging.md

```diff
@@ -0,0 +1,322 @@
+but this will provide more driver-specific info as well such as:
+  * What drivers are found
+  * Where they were found
+  * If there is any incompatibility with a given driver
+  * If any environment variables are disabling any of the drivers
+
+For example, the output of the loader looking for drivers on a Linux system may
+look like the following (NOTE: additional spaces have been removed from the
+output for easier reading):
+
+```
+DRIVER: Searching for driver manifest files
+DRIVER:    In following folders:
+DRIVER:       /home/$(USER)/.config/vulkan/icd.d
+DRIVER:       /etc/xdg/vulkan/icd.d
+DRIVER:       /etc/vulkan/icd.d
+DRIVER:       /home/$(USER)/.local/share/vulkan/icd.d
+DRIVER:       /home/$(USER)/.local/share/flatpak/exports/share/vulkan/icd.d
+DRIVER:       /var/lib/flatpak/exports/share/vulkan/icd.d
+DRIVER:       /usr/local/share/vulkan/icd.d
+DRIVER:       /usr/share/vulkan/icd.d
+DRIVER:    Found the following files:
+DRIVER:       /usr/share/vulkan/icd.d/intel_icd.x86_64.json
+DRIVER:       /usr/share/vulkan/icd.d/lvp_icd.x86_64.json
+DRIVER:       /usr/share/vulkan/icd.d/radeon_icd.x86_64.json
+DRIVER:       /usr/share/vulkan/icd.d/lvp_icd.i686.json
+DRIVER:       /usr/share/vulkan/icd.d/radeon_icd.i686.json
+DRIVER:       /usr/share/vulkan/icd.d/intel_icd.i686.json
+DRIVER:       /usr/share/vulkan/icd.d/nvidia_icd.json
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/intel_icd.x86_64.json, version "1.0.0"
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/lvp_icd.x86_64.json, version "1.0.0"
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/radeon_icd.x86_64.json, version "1.0.0"
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/lvp_icd.i686.json, version "1.0.0"
+DRIVER: Requested driver /usr/lib/libvulkan_lvp.so was wrong bit-type. Ignoring this JSON
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/radeon_icd.i686.json, version "1.0.0"
+DRIVER: Requested driver /usr/lib/libvulkan_radeon.so was wrong bit-type. Ignoring this JSON
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/intel_icd.i686.json, version "1.0.0"
+DRIVER: Requested driver /usr/lib/libvulkan_intel.so was wrong bit-type. Ignoring this JSON
+DRIVER: Found ICD manifest file /usr/share/vulkan/icd.d/nvidia_icd.json, version "1.0.0"
+```
+
+Then when the application selects the device to use, you will see the Vulkan
+device call chain reported in the following way (NOTE: additional spaces have
+been removed from the output for easier reading):
+
+```
+DRIVER: vkCreateDevice layer callstack setup to:
+DRIVER:    <Application>
+DRIVER:      ||
+DRIVER:    <Loader>
+DRIVER:      ||
+DRIVER:    <Device>
+DRIVER:        Using "Intel(R) UHD Graphics 630 (CFL GT2)" with driver: "/usr/lib64/libvulkan_intel.so"
+```
+
+
+### Selectively Enable Specific Drivers
+
+You can now use the filtering environment variables
+(`VK_LOADER_DRIVERS_SELECT` and `VK_LOADER_DRIVERS_DISABLE`) to selectively
+select or disable various drivers.
```

Comment:
`selectively enable or disable`, rather than `selectively select`?

Reply:
No, the drivers are enabled by default.  The process for drivers is select (i.e. force a particular driver) and disable.

Reply:
Ah good point...  How about:

```
You can now use the filtering environment variables
(`VK_LOADER_DRIVERS_SELECT` and `VK_LOADER_DRIVERS_DISABLE`) to control what
drivers the loader discovers.
```

Reply:
```
You can now use the filtering environment variables
(`VK_LOADER_DRIVERS_SELECT` and `VK_LOADER_DRIVERS_DISABLE`) to control what
drivers the loader will attempt to load.
```
maybe? Otherwise your version is fine.